lib/error-format.ts

```diff
@@ -0,0 +1,33 @@
+export function formatCallGraphError(error: Error): string {
+  if (error.message === 'Could not find target folder') {
+    return 'Couldn\'t scan for reachable vulns, to do so your code should be compiled, please run `mvn compile`.';
```

Comment:
Failed to scan for reachable vulns. Compile your code by executing `mvn compile` and try again, please.

What you think of this option?

Reply:
How about: "Failed to scan for reachable vulns. Please compile your code by running `mvn compile`"?

Reply:
Thanks, I'm combining both of your suggestions